.github/workflows/api-release.yml

```
@@ -39,0 +1,81 @@
+  release-maven:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    outputs:
+      MVN_ARTIFACT_ID: ${{ steps.maven-release-step.outputs.MVN_ARTIFACT_ID }}
+      ARTIFACT_NAME: ${{ steps.maven-release-step.outputs.artifact-name }}
+    steps:
+      - name: Maven build and release
+        id: maven-release-step
+        uses: it-at-m/lhm_actions/action-templates/actions/action-maven-release@b2b49a8b4344e3b5ac90dd1ee9a0c59860c41383 # v1.0.16
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)